src/Button.php

```diff
@@ -57,6 +56,44 @@ public static function link(
         return self::widget($constructorArguments, $config, $theme)->label($label)->type(ButtonType::LINK)->url($url);
     }
 
+    /**
+     * Get an instance of a reset button input.
+     *
+     * @param string|Stringable $value The content of the button. By default, it's "Reset".
+     * @param array $constructorArguments The constructor arguments.
+     * @param array $config The configuration.
+     * @param string|null $theme The theme.
+     *
+     * @return self A new instance with the input of "reset" type.
+     */
+    public static function resetInput(
```

Comment:
For consistency factory methods and enum cases should be equal. Currently, these are different: `resetInput()` and `INPUT_RESET`.

Reply:
done.